spec/javascripts/components/feedback-spec.js

```diff
@@ -106,6 +105,20 @@ describe("Feedback component", function () {
     expect($('#something-is-wrong').find("[name=referrer]").val()).toBe("unknown");
   });
 
+  it("should append a hidden 'ga_client_id' field to the form", function() {
+    loadFeedbackComponent();
+    $('a.js-page-is-not-useful').click();
```

Comment:
Should be able to target this without the `a` in the selector.

Reply:
👍